rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -687,7 +689,7 @@ object Reduce {
                                       .map(
                                         keyValuePairs => GString(interpolate(lhs, keyValuePairs))
                                       )
-                           _ <- costAccountingAlg.charge(LOOKUP_COST * lhs.length)
+                           _ <- costAccountingAlg.charge(LOOKUP_COST * (lhs.length * rhs.size))
```

Comment:
This type of change feels like it should have a test. Also the .length vs .size difference seems to stand out - there is no different right?

Reply:
Also why is this multiplicative versus the prepending costs additive?

Reply:
Oh, it should be additive. It was late, thanks for catching that.

Reply:
Because there's no `length` on `Map`.